DyldExtractor/Converter/ObjCConvertor.py

```diff
@@ -75,11 +75,22 @@ def convert(self) -> None:
 	def processSegments(self) -> None:
 		segments: List[MachO.segment_command_64] = self.machoFile.getLoadCommand(MachO.LoadCommands.LC_SEGMENT_64, multiple=True)
 
+		# Segments we need to zero out a byte on
```

Comment:
This should be unnecessary, the extra byte in the pointer is for rebase data. This should have been taken care of by using RebaseConverter.